packages/common/src/events/edit.ts

```diff
@@ -1,10 +1,11 @@
 import { IHubEvent } from "../core/types/IHubEvent";
-import { IHubRequestOptions } from "../types";
+import { IHubRequestOptions, IHubUserRequestOptions } from "../types";
```

Comment:
Can remove `IHubUserRequestOptions` from this import statement as it's now unused